entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-htmlproofer "${INPUT_DIRECTORY}"
\ No newline at end of file
+htmlproofer ${INPUT_ARGUMENTS:-""} "${INPUT_DIRECTORY}"
```

Comment:
No need to set a default if it won't crash (see [my previous comment](https://github.com/chabad360/htmlproofer/pull/2#discussion_r347686805) for more info).

Reply:
Done